cmd/commands/rook.go

```diff
@@ -47,7 +47,7 @@ var versionCmd = &cobra.Command{
 
 var purgeCmd = &cobra.Command{
 	Use:   "purge-osd",
-	Short: "Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs.",
+	Short: "Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs. For example: kubectl rook-ceph rook purge-osd 0,1",
```

Comment:
I might suggest a slightly simpler example for the CLI help, but I'm ok if you want to keep your text too.
```suggestion
	Short: "Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs, for example, purge-osd 0,1",
```

Reply:
updated, will merge after the CI